keyboards/ebastler/e80_1800/readme.md

```diff
@@ -0,0 +1,25 @@
+# E80-1800
+
+![E80_1800_sample_image](https://github.com/ebastler/E80-1800/blob/master/img/bottom.png)
```

Comment:
please rehost this on an image hosting site, such as imgur

Reply:
Okay, changed to imgur